charts/lfx-v2-project-service/templates/nats-kv-buckets.yaml

```diff
@@ -0,0 +1,40 @@
+# Copyright The Linux Foundation and each contributor to LFX.
+# SPDX-License-Identifier: MIT
+---
+{{- if .Values.nats.kv_bucket_project_base.creation }}
+apiVersion: jetstream.nats.io/v1beta2
+kind: KeyValue
+metadata:
+  name: {{ .Values.nats.kv_bucket_project_base.name }}
+  namespace: lfx
```

Comment:
I've been using `{{ .Release.Namespace }}` here, which then installs it to the same namespace that the chart is installed to.